qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -2850,14 +2871,27 @@ wait_for_group_replay_stopped()
     wait_for_group_replay_state "${cluster}" "${group_spec}" 'stopped' "${image_count}" 'false'
 }
 
-get_newest_group_snapshot_id()
+get_newest_user_group_snapshot_id()
```

Comment:
This function does not always get the newest user group snapshot's ID. It doesn't filter for user group snapshots. Currently, it get the newest group snapshot that has been created. As the function body is currently, naming the function `get_newest_created_group_snapshot_id()` makes more sense to me.